crates/hyperqueue/src/client/commands/event/mod.rs

```diff
@@ -30,12 +38,37 @@ struct ExportOpts {
     logfile: PathBuf,
 }
 
-pub fn command_event_log(opts: EventLogOpts) -> anyhow::Result<()> {
+pub async fn command_event_log(
+    gsettings: &GlobalSettings,
+    opts: EventLogOpts,
+) -> anyhow::Result<()> {
     match opts.command {
         EventCommand::Export(opts) => export_json(opts),
+        EventCommand::Stream => stream_json(gsettings).await,
     }
 }
 
+async fn stream_json(gsettings: &GlobalSettings) -> anyhow::Result<()> {
+    let mut connection = get_client_session(gsettings.server_directory()).await?;
+    connection
+        .connection()
+        .send(FromClientMessage::StreamEvents)
+        .await?;
+    let stdout = std::io::stdout();
+    let stdout = stdout.lock();
+    let mut stdout = BufWriter::new(stdout);
+    while let Some(event) = connection.connection().receive().await {
+        let event = event?;
+        if let ToClientMessage::Event(e) = event {
+            writeln!(stdout, "{}", format_event(e))?;
+            stdout.flush()?;
```

Comment:
Maybe we could flush only every once in a while? Flushing after every event kind of defeats the purpose of the `BufWriter`.

Reply:
BufWriter still have purpose as an event may be written in several writes (especially large event "submit").
However, I consider this mostly as command for testing. As we do not have non-testing users for this, it is hard to find the right flush policy.

Reply:
Ok, fair enough.